src/directives/popover.js

```diff
@@ -53,8 +54,7 @@ ngeo.popoverDirective = function() {
 
       if (attrs['ngeoPopoverDismiss']) {
         $(attrs['ngeoPopoverDismiss']).on('scroll', function() {
-          ngeoPopoverCtrl.anchorElm.popover('hide');
-          ngeoPopoverCtrl.shown = false;
+          ngeoPopoverCtrl.dismissPopover();
```

Comment:
It works but I haven't added the "ngeo-popover-dissmiss" attribut in the layertree.html. IMO it should be an other PR to fix "dismiss popover when layertree is scrolled"